pkg/cmd/step/pr/step_pr_comment.go

```diff
@@ -59,15 +61,27 @@ func NewCmdStepPRComment(commonOpts *opts.CommonOptions) *cobra.Command {
 
 // Run implements this command
 func (o *StepPRCommentOptions) Run() error {
+	if o.Flags.PR == "" {
+		o.Flags.PR = os.Getenv("PULL_NUMBER")
```

Comment:
We seem to be more and more setting options via environment variables. Instead of doing this ad hoc I am wondering whether a more generic approach makes more sense using, for example, Cobra's ability to automatically bind environment variables to options. The benefit would be a more consistent approach to option handling.

Reply:
@garethjevans ^^